db/migrate/20230620113412_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :bio
+      t.integer :posts_counter
```

Comment:
- [ ] [REQUIRED] Good job so far! You forgot to create `t.string :photo` as photo should be a link to an image not a file 🙏🏼